pom.xml

```diff
@@ -59,9 +59,9 @@
     <svm.version>24.1.1</svm.version>
     <micrometer.version>1.13.8</micrometer.version>
     <blockhound.version>1.0.10.RELEASE</blockhound.version>
-    <testcontainers.version>1.20.3</testcontainers.version>
+    <testcontainers.version>1.20.4</testcontainers.version>
     <archunit-junit5.version>1.3.0</archunit-junit5.version>
-    <build-resources.version>5.18.1</build-resources.version>
+    <build-resources.version>2024-11.1</build-resources.version>
```

Comment:
Looks like someone else went to cal versioning recently as well...

Reply:
This project got moved and has different versioning.